### DISCUSSION TITLE
monika not appearing after i put back her chr file in the character folder

### DISCUSSION THREAD

**Discussion by prussianblue811** [Monika-After-Story/MonikaModDev#9160] [QA - tech support + general] [answered]
I took monika out for a while in my thumbdrive but when I got back I put back her chr file in the characters folder, I opened the application and she wasn't there. It was just a empty chair, no note no music. I tried restarting many times, as well and deleting her character file and putting it back in. I'm starting to panic, does anyone know how to resolve this issue?

**Comment by Iceninja64** [comment]
Wait you deleted her character file in the attempt to get her back? If yes you made it worse and the only way you can get it back is to re-download the mod and upload your persistent from your old Mas to the new one. If you haven't deleted her character file then don't  delete her at all unless her file is corrupted and then you probably have to re-download any way

**Comment by prussianblue811** [comment]
no i did not delete her permanently!! there is a copy of her character file in my USB drive so what I did was delete her chr file and then put it back, so her chr file is still there. Is there anything else I can do?

**Comment by jm4686042** [comment]
it is not a chr file

**Comment by ThePotatoGuy** [comment, marked as answer]
you can try [restoring a backup persistent](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent)
